index.html

```diff
@@ -0,0 +1,68 @@
+<!--HTML page created for learning purposes-->
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <title id="title">Geena's practice HTML file</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        
+        <div class="banner note">
+          <h1>Using paragraphs</h1>
+          <p>Just testing a div p </p>
+        </div>
+        <p>
+            This page is for practicing on using the following:
+            <ol>
+                <li class="para">Paragraphs</li>
+                <li class="links">Links</li>
+                <li class="ima">Images</li>
+                <li>Spans</li>
+                <li>Tables</li>
+                <li>Lists</li>
+            </ol>
+        </p>
+        <h2 class="links">Using links</h2>
+        <a href="https://www.google.com" target="_top"> This is an example link to Google. 
+          <br>
+        This is an example link to <a href="https://www.yahoo.com" target="_blank"> Yahoo. </a>
+        <h3>Using spans</h3>
+        <p> This is the color <span style="color:red;font-weight:bold">red</span> using span. </p>
+        <h4 class="ima">Using images</h4>
+        <p><img id="cat" src="files/cat.jpg" style="width: 200px" alt="A picture of some code"> </p>
+        <h5>Using tables</h5>
+        <table>
+            <tr>
+              <th>Col 1</th>
+              <th>Col 2</th>
+            </tr>
+            <tr>
+              <td>Item 2</td>
+              <td>Item 1</td>
+            </tr>
+          </table>
+          <h6> Using lists</h6>
+          <p>Here is a list of ordered items:</p>
+            <ul>
+                <li>Uno</li>
+                <li>Dos</li>
+                <li>Tres</li>
+            </ul>
+            Search: <input type="text" name="search">
+<br>
+            
+              <!--
+
+              -->
+              This is exercise 1:
+              <div class="note">
+                <p id="first">This is a paragraph inside a div element.</p>
+                <p>This is another paragraph inside a div element.</p>
+                <span><p>This a paragraph inside a span element, inside a div element.</p></span>
```

Comment:
I generally prefer to use CSS classes to achieve inline elements instead of using ```<span>```.  When using multiple elements, I prefer separate rows for readability, or to achieve that span effect, I use CSS to avoid extra elements.
```
<span>
    <p>This is a.....div element.</p>
</span>
```